seawatch_registration/tests/test_urls.py

```diff
@@ -45,3 +46,8 @@ def test_urls_skill_update(self):
     def test_urls_question_update(self):
         url = reverse('question_update')
         self.assertEquals(resolve(url).func.__name__, question.UpdateView.as_view().__name__)
+
+    def test_urls_availabilities(self):
```

Comment:
Wieso nicht auch für die NoJS Alternative?

Reply:
hinzugefügt in 9c10458